docs/ReleaseGuide.md

```diff
@@ -1,25 +1,25 @@
 ## Guide to release new version of python-bindings
 The developer who is releasing a new version of the python-bindings is expected to follow this work flow:
 
-The release of the `python-bindings` repository is made directly from a release branch called `python-bindings-v2.1.1.1`. This branch is mainly needed to help other developers with testing.
+The release of the `python-bindings` repository version `2.x.y.z` is made directly from a release branch called `python-bindings-v2.x.y.z`. This branch is mainly needed to help other developers with testing.
 
-1. Create a branch called `python-bindings-v2.1.1.1` from the latest commit of the `develop` branch.
+1. Create a branch called `python-bindings-v2.x.y.z` from the latest commit of the `develop` branch.
 
-2. [Open a Pull Request `master` <-- `python-bindings-v2.1.1.1`](https://github.com/precice/python-bindings/compare/master...master) named after the version (i.e. `Release v2.1.1.1`) and briefly describe the new features of the release in the PR description.
+2. [Open a Pull Request `master` <-- `python-bindings-v2.x.y.z`](https://github.com/precice/python-bindings/compare/master...master) named after the version (i.e. `Release v2.x.y.z`) and briefly describe the new features of the release in the PR description.
 
 3. Bump the version in the following places:
 
-    * `CHANGELOG.md` on `python-bindings-v2.1.1.1`.
+    * `CHANGELOG.md` on `python-bindings-v2.x.y.z`. For this change pushing directly to the release branch is permissible.
     * There is no need to bump the version anywhere else, since we use the [python-versioneer](https://github.com/python-versioneer/python-versioneer/) for maintaining the version everywhere else.
 
-4. [Draft a New Release](https://github.com/precice/python-bindings/releases/new) in the `Releases` section of the repository page in a web browser. The release tag needs to be the exact version number (i.e.`v2.1.1.1` or `v2.1.1.1rc1`, compare to [existing tags](https://github.com/precice/python-bindings/tags)). Use `@target:master`. Release title is also the version number (i.e. `v2.1.1.1` or `v2.1.1.1rc1`, compare to [existing releases](https://github.com/precice/python-bindings/tags)). 
+4. [Draft a New Release](https://github.com/precice/python-bindings/releases/new) in the `Releases` section of the repository page in a web browser. The release tag needs to be the exact version number (i.e.`v2.1.1.1` or `v2.1.1.1rc1`, compare to [existing tags](https://github.com/precice/python-bindings/tags)). Use `@target:master`. Release title is also the version number (i.e. `v2.x.y.z` or `v2.x.y.zrc1`, compare to [existing releases](https://github.com/precice/python-bindings/tags)). 
```

Comment:
I find the notation `v2.x.y.zrc1` a bit confusing.

Reply:
Additionally: It should be `vw.x.y.zrca` to be consistent (which is totally unreadable). Why not use capital letters? ` vW.X.Y.ZrcA` but the `W` and `A` are a bit strange. Probably due to our non-standard versionin scheme...